namespaces/live.cloud-platform.service.justice.gov.uk/visit-someone-in-prison-backend-svc-dev/resources/irsa.tf

```diff
@@ -1,8 +1,34 @@
-module "app-irsa" {
-  source               = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=2.0.0"
+# Add the names of the SQS/SNS which the app needs permissions to access.
+# The value of each item should be the namespace where the SQS/SNS was created.
+# This information is used to collect the IAM policies which are used by the IRSA module.
+locals {
+  sqs_queues = {
+    "Digital-Prison-Services-dev-hmpps_prison_visits_event_queue" = "hmpps-domain-events-dev",
+    "Digital-Prison-Services-dev-hmpps_prison_visits_event_dlq" = "hmpps-domain-events-dev",
+  }
+  sns_queues = {
+    "Digital-Prison-Services-dev-hmpps_domain_events_dev"    = "hmpps-domain-events-dev",
```

Comment:
the topic name should be `cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573`